src/Components/Login.js

```diff
@@ -1,16 +1,43 @@
 import React, {Component} from 'react'
 import '../CSS/Login.css';
+import App from '../App';
 
 class Login extends Component {
   constructor(){
     super();
     this.state = {
     username:'',
-    password: ''
+    password: '',
+    error: ''
```

Comment:
Now that we're moving a substantial part of login to app in the following changes, I'm wondering how else login would change? Namely, class component to functional.